react/components/Portal.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { FunctionComponent } from 'react'
+import ReactDOM from 'react-dom'
+
+export interface Props {
+  target?: HTMLElement
+  zIndex?: number
```

Comment:
frescurite: since you are defining a default value, this isn't optional

Reply:
@jgfidelis I tried doing that, but it actually complains about it, so 🤷‍♂